### PR TITLE
Styx integration

### PR DESCRIPTION
**Pull request by Slion**
See: Slion/Fulguris#265

### CODE REVIEW

**Review by Slion — COMMENTED** [review]
Nothing here.

File: app/src/main/java/acr/browser/lightning/browser/tabs/TabsDesktopAdapter.kt [hunk 1]
```diff
@@ -132,7 +132,7 @@ class TabsDesktopAdapter(
             viewHolder.iCardView.isChecked = false
             viewHolder.iCardView.isCheckable = false
             // Background tab
-            TextViewCompat.setTextAppearance(viewHolder.txtTitle, if (tab.isFrozen) R.style.italicText else R.style.normalText)
+            TextViewCompat.setTextAppearance(viewHolder.txtTitle, R.style.italicText)
```
Comment (Slion) [inline app/src/main/java/acr/browser/lightning/browser/tabs/TabsDesktopAdapter.kt]: This was actually disabling that feature and always using italic for background tabs regardless of whether they are frozen.

Comment (jamal2362) [reply app/src/main/java/acr/browser/lightning/browser/tabs/TabsDesktopAdapter.kt]: So this Italic font as it was before didn't work for me, for whatever reason. Therefore the change.

Comment (Slion) [reply app/src/main/java/acr/browser/lightning/browser/tabs/TabsDesktopAdapter.kt]: Works just fine here. Only frozen tabs are displayed using italic. Loaded background tabs are using normal fonts.

**Pull request merged**